ghost/admin/app/templates/settings/labs.hbs

```diff
@@ -226,19 +226,6 @@
                         </div>
                     </div>
                 </div>
-                <div class="gh-expandable-block">
-                    <div class="gh-expandable-header">
-                        <div>
-                            <h4 class="gh-expandable-title">Member Attribution</h4>
-                            <p class="gh-expandable-description">
-                               Track which posts & pages are driving Member Subscriptions & Paid Conversions
-                            </p>
-                        </div>
-                        <div class="for-switch">
-                           <GhFeatureFlag @flag="memberAttribution" />
-                        </div>
-                    </div>
-                </div>
```

Comment:
Can't we keep it here for local debugging reasons? As this is only visible when developer experiments is on

Reply:
Good shout